src/core/public/entry_point.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { CoreSystem } from './core_system';
+
+const injectedMetadata = JSON.parse(
+  document.querySelector('kbn-injected-metadata')!.getAttribute('data')!
+);
```

Comment:
Nah I just didn't bother moving it yet while I made sure this approach was sound. I'll add it before putting this up for review.

Reply:
Added back in. Since we're no longer using a dynamic template for the entrypoint, I'm using `if (process.env)` checks to make sure the APM module is only included in certain cases.